package ExceptionHandling;

public class TryCatch01 {
    public static void main(String[] args) {
//        try {
//            // 예외가 발생할 가능성이 있는 문장
//        } catch (Exception e1) {
//            // 첫 번째 예외가 발생 했을 경우 이를 처리하기 위해 실행될 문장
//        } catch (Exception1 e2) {
//            // 두 번째 예외가 발생 했을 경우 이를 처리하기 위해 실행될 문장
//        } catch (Exception2 e3) {
//            // 세 번째 예외가 발생 했을 경우 이를 처리하기 위해 실행될 문장
//        }
    }
}
